Luke/views.py

```diff
@@ -22,27 +26,47 @@ def add_req(request):
     api = get_api()
 
     if request.method == "POST":
-        result = api.handle_new_request(request.POST.get("request"))
-        if result:
-            return HttpResponse(content=result, status=200)
-        else:
-            return HttpResponse(content="invalid request", status=500)
+        # result = api.handle_new_request(request.POST.get("request"))
+        result = api.handle_new_request(req=request)
+        return check_result(result)
```

Comment:
we need to add elif reqquest.method == "GET", or remove get from the require_http_methods decorator
I would tend to say we should add elif, and return a list of all the requests in the db